var/spack/repos/builtin/packages/aws-parallelcluster/package.py

```diff
@@ -33,13 +34,17 @@ class AwsParallelcluster(PythonPackage):
 
     depends_on('python@2.7:', type=('build', 'run'))
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-boto3@1.10.15:', type=('build', 'run'))
-    depends_on('py-future@0.16.0:0.18.2', type=('build', 'run'))
-    depends_on('py-tabulate@0.8.2:0.8.3', type=('build', 'run'))
+    depends_on('py-boto3@1.10.15:', when='@:2.8', type=('build', 'run'))
+    depends_on('py-boto3@1.14.3:', when='@2.9:', type=('build', 'run'))
+    depends_on('py-future@0.16.0:0.18.2', when='@:2.8', type=('build', 'run'))
+    depends_on('py-future@0.18.2:', when='@2.9:', type=('build', 'run'))
```

Comment:
It looks like this version requirement didn't change in 2.9.0. Are you looking at master instead of 2.9.0?